_config/config.yml

```diff
@@ -25,3 +25,10 @@ Symbiote\GridFieldExtensions\GridFieldAddNewMultiClassHandler:
 SilverStripe\Core\Injector\Injector:
   SilverStripe\CMS\Controllers\CMSSiteTreeFilter_Search:
     class: DNADesign\Elemental\Controllers\ElementSiteTreeFilterSearch
+  Psr\SimpleCache\CacheInterface.ElementTabCache:
+    factory: SilverStripe\Core\Cache\CacheFactory
+    constructor:
+      namespace: "ElementTabCache"
+  DNADesign\Elemental\Services\ElementTabProvider:
+    properties:
+      cache: %$Psr\SimpleCache\CacheInterface.ElementTabCache
```

Comment:
I think it's always safest to quote injector definitions like this, I've had problems without quoting them before. Also we could put these config definitions into a new cache.yml file